build/release-notes.md

```diff
@@ -8,6 +8,127 @@ toc_max: 2
+* Detect invalid attestation configuration. {% include github_issue.md repo="docker/buildx" number="1545" %}
+* Update containerd patches to fix possible push regression affecting
+  `imagetools` commands. {% include github_issue.md repo="docker/buildx" number="1559" %}
+
+## 0.10.0
+
+{% include release-date.html date="2023-01-10" %}
+
+> **Note**
+>
+> Buildx v0.10 enables support for a minimal [SLSA Provenance](https://slsa.dev/provenance/)
+> attestation, which requires support for [OCI-compliant](https://github.com/opencontainers/image-spec)
+> multi-platform images. This may introduce issues with registry and runtime support
+> (e.g. [Google Cloud Run and AWS Lambda](https://github.com/docker/buildx/issues/1533)).
+> You can optionally disable the default provenance attestation functionality
+> using `--provenance=false`.
+
+### New
+
+* Build command supports new flags `--attest` and shorthands `--sbom` and
+  `--provenance` for adding attestations for your current build.
+  `--attest type=sbom` or `--sbom=true` will generate SBOM (Software Bill of
+  Materials) attestation for your build result in SPDX format.
+  `--attest type=provenance` or `--provenance=true` will generate [SLSA provenance attestation](attestations/slsa-provenance.md)
+  for your build result with information about how the build was performed.
+  When creating OCI images, a minimal provenance attestation is included with
+  the image by default. This feature requires BuildKit v0.11.0+. Read more
+  about advanced [SBOM options in our dedicated docs](attestations/sbom.md). {% include github_issue.md repo="docker/buildx" number="1412" %}
+  {% include github_issue.md repo="docker/buildx" number="1475" %}
+* When building with BuildKit that supports provenance attestations Buildx will
+  automatically share the version control information of your build context, so
+  it can be shown in provenance for later debugging. Previously this only
+  happened when building from a Git URL directly. To opt-out of this behavior
+  you can set `BUILDX_GIT_INFO=0`. Optionally you can also automatically define
+  labels with VCS info by setting `BUILDX_GIT_LABELS=1`. {% include github_issue.md repo="docker/buildx" number="1462" %}
+  {% include github_issue.md repo="docker/buildx" number="1297" %} {% include github_issue.md repo="docker/buildx" number="1341" %}
+  {% include github_issue.md repo="docker/buildx" number="1468" %} {% include github_issue.md repo="docker/buildx" number="1477" %}
+* Named contexts with `--build-context` now support `oci-layout://` protocol
+  for initializing the context with a value of a local OCI layout directory.
+  E.g. `--build-context stagename=oci-layout://path/to/dir`. This feature
+  requires BuildKit v0.11.0+ and Dockerfile 1.5.0+. {% include github_issue.md repo="docker/buildx" number="1456" %}
+* Bake now supports [resource interpolation](bake/configuring-build.md#resource-interpolation)
```

Comment:
I am unsure if we should refer to Bake as "Bake" or as the command `buildx bake` - it almost seems bigger than a command. But other things like `imagetools` we do refer to as commands.

Reply:
I'm not sure, here it's about the Bake definition, not really the bake command.